tests/unit/lms/services/annotation_activity_email_test.py

```diff
@@ -127,6 +127,36 @@ def test_with_should_not_notify_unsubscribed(
 
         send.delay.assert_not_called()
 
+    def test_with_should_not_notify_feature_disabled(
```

Comment:
There was actually a coverage gap due to checking both conditions on the same if.